diff -rup lohit-malayalam-2.4.4/Lohit-Malayalam.sfd lohit-malayalam-2.4.4_mod/Lohit-Malayalam.sfd
--- lohit-malayalam-2.4.4/Lohit-Malayalam.sfd	2009-09-21 16:14:28.000000000 +0530
+++ lohit-malayalam-2.4.4_mod/Lohit-Malayalam.sfd	2011-07-05 14:49:39.207229357 +0530
@@ -20,7 +20,7 @@ OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1159524182
-ModificationTime: 1253263612
+ModificationTime: 1309857574
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -351,7 +351,7 @@ DisplaySize: -24
 AntiAlias: 1
 FitToEm: 1
 WinInfo: 0 42 9
-BeginChars: 65725 300
+BeginChars: 65725 301
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -10412,7 +10412,7 @@ StartChar: glyph162
 Encoding: 65619 -1 167
 Width: 705
 GlyphClass: 2
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -19746,5 +19746,37 @@ SplineSet
  -311 -33 -311 -33 -229 -34 c 0,21,22
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 300
+Width: 624
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 499 m 0,0,1
+ 359 508 359 508 359 513 c 1,2,-1
+ 59 513 l 1,3,-1
+ 85 576 l 1,4,-1
+ 346 576 l 1,5,6
+ 340 588 340 588 331 598 c 0,7,8
+ 280 653 280 653 165 653 c 2,9,-1
+ 59 653 l 1,10,-1
+ 85 716 l 1,11,-1
+ 583 716 l 1,12,-1
+ 557 653 l 1,13,-1
+ 367 653 l 1,14,15
+ 406 630 406 630 420 576 c 1,16,-1
+ 583 576 l 1,17,-1
+ 557 513 l 1,18,-1
+ 429 513 l 1,19,20
+ 429 357.613 429 357.613 171 339 c 1,21,-1
+ 507 -1 l 1,22,-1
+ 407 -1 l 1,23,-1
+ 70 336 l 1,24,-1
+ 70 395 l 1,25,26
+ 345.891 393 345.891 393 358 499 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
